app/controllers/users_controller.rb

```diff
@@ -62,6 +62,64 @@ def filters
     end
   end
 
+  def change_role
+    add_breadcrumb @user.name, :user_path
```

Comment:
On the edit page of user, below name field, there should be a role field. In our project only admins and account owner can edit user, hence we will use the same form as before and just give a new role dropdown. Staff cannot edit user hence staff cannot change role or anything.